src/zenml/models/v2/core/artifact_version.py

```diff
@@ -139,33 +139,11 @@ def str_field_max_length_check(cls, value: Any) -> Any:
         )
         return value
 
-    @model_validator(mode="after")
```

Comment:
Any reason that is not needed anymore?

Reply:
No reason. I was over-eager and removed it by mistake.